zebedee/client.go

```diff
@@ -369,9 +369,7 @@ func (c *Client) GetCollection(ctx context.Context, userAccessToken, collectionI
 }
 
 // GetBulletin retrieves a bulletin from zebedee
-func (c *Client) GetBulletin(ctx context.Context, userAccessToken, lang, uri string) (Bulletin, error) {
-	collectionID := ""
-
+func (c *Client) GetBulletin(ctx context.Context, userAccessToken, collectionID, lang, uri string) (Bulletin, error) {
```

Comment:
just wondering what will happen if call is made with collectionID, should we include that scenario in the test too

Reply:
Thanks @bpathak-ons, that's a good point and I did consider passing a non empty string in the test but decided not to for simplicity. 
When we have a collection id, the zebedee URL [is prefixed with that value](https://github.com/ONSdigital/dp-api-clients-go/blob/9bc45000a35d64f1d14604377114892ba762b38c/zebedee/client.go#L440-L442). The current tests are mocking an http server whose [serving routes](https://github.com/ONSdigital/dp-api-clients-go/blob/9bc45000a35d64f1d14604377114892ba762b38c/zebedee/client_test.go#L34) could be easily amended to accept this new url but asserting the values wouldn't be straightforward. Especially since none of the other existing tests currently do anything around that and that change would mean a larger diff. 
I will be happy to look into it if we think there is value in it?